src/ru/skypro/Main.java

```diff
@@ -1,8 +1,44 @@
 package ru.skypro;
 
 public class Main {
-
     public static void main(String[] args) {
-	// write your code here
+        Task1();
+        Task2();
+        Task3();
+    }
+    public static void Task1()
+    {
+        System.out.println("Первое задание. Числа");
+        int i = 1;
+        while (i<=10)
+        {
+            System.out.print(i+" ");
+            i++;
+        }
+        System.out.println();
+        for(i--;i>=1;i--)
+        {
+            System.out.print(i+" ");
+        }
+        System.out.println();
+    }
+    public static void Task2() {
+        System.out.println("Второе задание. Пятница - отчётный день");
+        int Friday = 1;
+        int DaysinMonth = 31;
```

Comment:
переменные именуются с прописной буквы
```suggestion
        int friday = 1;
        int daysInMonth = 31;
```